docker-with-buildargs/index.js

```diff
@@ -0,0 +1,18 @@
+const express = require("express");
+
+const app = express();
+const port = 8080;
+
+app.get("/hello-world", (req, res) => {
+  res.status(200).send("hello-world");
+});
+
+app.get("/env", (req, res) => {
+  res.status(200).json(process.env);
+});
+
+app.get("/cmd", (req, res) => {
+  res.status(200).json(process.argv);
+});
```

Comment:
At the moment the test is only verifying that the image builds successfully. It would be nice to create an endpoint which returns build-args, that way we can curl this endpoint and verify the values in the test.

Reply:
I don't think this is an issue, since I am currently setting the image tag with the build arg. So the image will never build if the build arg is not passed properly ;)
We should not test more functionality, than we need to test here.